src/Repository.php

```diff
@@ -179,7 +180,7 @@ public function updateModel(array $data, Model $model)
      *
      * @param int $perPage
      * @param array $columns
-     * @return \Illuminate\Pagination\AbstractPaginator
+     * @return \Illuminate\Contracts\Pagination\LengthAwarePaginator
```

Comment:
It appears that it is no longer the case in Laravel 5.3 — the paginate method always returns an instance of \Illuminate\Contracts\Pagination\LengthAwarePaginator. I’ll bump the Laravel version up instead.